HOWTO.md

```diff
@@ -0,0 +1,118 @@
+# 📬 Submission Guide
+
+> Travelled recently? Have pictures to share? You're in the right place!
+
+## 🖇️ Forking the repository
+
+**1.** Go to [CeSIUMpeloMundo](https://github.com/cesium/CeSIUMpeloMundo)
+
+**2.** In the top right corner, click on **Fork**
+
+## 🔗 Cloning the forked repository to your machine
+
+**1.** Go to _your fork_ and click **Code**
+
+> **Note**  
+> The link to your forked repository should be `https://github.com/<your-username>/CeSIUMpeloMundo`
+
+**2.** Copy the given HTTPS link
+
+**3.** On your terminal, run the following command:
+
+```
+git clone <https-link>
+```
+
+> **Note**  
+> If you'd rather clone with SSH, copy the SSH key and run `git clone <ssh-key>`
+
+**Now you have a copy of the repository you can work with.**
+
+## ⛓️ Creating a branch
```

Comment:
After this step, I think we could link to `CONTRIBUTING.md` to make the contributor aware that they can get a local preview.